packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -27,7 +28,7 @@ export class Snapshot {
       deleteMissingResources: false, // TODO: CDX-361: Add flag to support missing resources deletion
     });
 
-    await this.waitUntilDone();
+    await this.waitUntilDone(ResourceSnapshotsReportType.DryRun);
```

Comment:
Making the wait more robust by specifying the operation to wait for

Reply:
In that case, I would rename the method accordingly to be explicit:
`waitUntilOperationIsDone(operationType: ResourceSnaphostReportType)`